src/memory.cpp

```diff
@@ -158,7 +158,7 @@ cvk_sampler::create(cvk_context* context, bool normalized_coords,
     return sampler.release();
 }
 
-bool cvk_sampler::init() {
+bool cvk_sampler::init(bool norm) {
```

Comment:
s/norm/force_normalized_coordinates/ (or better name). I think we should use a more descriptive name.

Reply:
Done